pkgs/top-level/python-packages.nix

```diff
@@ -23151,6 +23151,26 @@ in {
     };
   };
 
+  typeguard = buildPythonPackage rec {
```

Comment:
Due the size of this file, we add all new python packages to python-modules and only reference them here.